tests/test_trtllm_gen_attention.py

```diff
@@ -398,10 +398,32 @@ def test_trtllm_batch_prefill(
     else:
         rtol, atol = 1e-2, 1e-2
 
-    # convert to float32 for fp8 is not supported by assert_close
-    torch.testing.assert_close(
-        output.float() * o_scale, output_ref.float(), rtol=rtol, atol=atol
-    )
+    # Arbitary small mismatch rate
+    allowed_mismatch_rate = 1e-7
+
+    try:
+        # convert to float32 for fp8 is not supported by assert_close
+        torch.testing.assert_close(
+            output.float() * o_scale, output_ref.float(), rtol=rtol, atol=atol
+        )
+    except AssertionError as err:
+        # Allow for a small mismatch rate for low precision outputs
```

Comment:
Can we summarize a function for this kind of check (it will be useful especially for low-precision kernels), such as `flashinfer.testing.assert_close` with an parameter of `max_failed_ratio`